src/address.js

```diff
@@ -18,7 +18,7 @@ function fromBase58Check (address) {
 function toBase58Check (hash, version) {
   typeforce(types.tuple(types.Hash160bit, types.UInt8), arguments)
 
-  var payload = new Buffer(21)
+  var payload = Buffer.allocUnsafe(21)
```

Comment:
@fanatid I've used `Buffer.alloc` all through the tests... and `Buffer.allocUnsafe` in parts of the code where were previously using `new Buffer`; so no change.

We should probably only need need `Buffer.alloc` in cases of user input no?